templates/register.html

```diff
@@ -38,8 +39,10 @@ <h2>We're looking forward to meeting you!</h2>
                 <img src="/images/mitosis-lab-4.webp" width="500" height="380" alt="">
                 <div class="col">
                     <h2>Venue</h2>
-                    <p>The Matrix Conference will happen at [Kaleidoscoop](https://www.kaleidos.coop/) in Strasbourg, France.</p>
-                    <p>Please <a href="mailto:conference@foundation.matrix.org">get in contact</a> if you are interested in sponsoring.</p>
+                    <p>The Matrix Conference will happen at [Kaleidoscoop](https://www.kaleidos.coop/) in Strasbourg,
```

Comment:
md does not work here, oops

Reply:
Fixed in [7732feb](https://github.com/matrix-org/matrix-conf-website/pull/103/commits/7732feb9843e006dec67bd68dec59089222649fe)